### ISSUE TITLE
Submit button is not showing under main menu when editing a request. 

### ISSUE THREAD

**Issue by CIManager** [WorkMAE/CEM-Integration#50]
Submit button is not showing under the main menu when editing an existing request.

**Comment by FourUnbelievableCoolKids** [comment]
It DOES appear, but only after the user changes tabs.

**Comment by WorkMAE** [comment]
Fixed and duplicate of issue already listed
